core/server/api/v2/utils/serializers/input/pages.js

```diff
@@ -3,6 +3,27 @@ const debug = require('ghost-ignition').debug('api:v2:utils:serializers:input:pa
 const converters = require('../../../../../lib/mobiledoc/converters');
 const url = require('./utils/url');
 const localUtils = require('../../index');
+const mongo = require('./utils/mongo');
+
+/*
+ * Replaces references of "page" in filters
+ * with the correct column "type"
+ */
+function replacePageWithType(mongoJSON) {
```

Comment:
Was thinking to write a comment to replace the duplicate structure in here and post serializer, but think the rule of 3 is good to follow :sweat_smile:

Reply:
:sweat_smile: Yeah I think to leave it for now - and see what other changes come about with future api versions!